### PR TITLE
New Library Perspective (new authoring)

### PR DESCRIPTION
**Pull request by ederign**

### CODE REVIEW

**Comment by manstis** [comment]
Jenkins please retest this.

File: drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java [hunk 1]
```diff
@@ -49,6 +51,12 @@
     protected WorkbenchMenuBarPresenter menuBar;
 
     @Inject
+    AdminPage adminPage;
```
Comment (ederign) [inline drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java]: done

File: drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java [hunk 2]
```diff
@@ -65,6 +73,13 @@ public DroolsWorkbenchEntryPoint( final Caller<AppConfigService> appConfigServic
 
     @Override
     public void setupMenu() {
+        adminPage.addScreen( "root", "Settings" );
```
Comment (ederign) [inline drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java]: done

File: drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java [hunk 3]
```diff
@@ -65,6 +73,13 @@ public DroolsWorkbenchEntryPoint( final Caller<AppConfigService> appConfigServic
...
+        adminPage.addPreference( "root",
+                                 "LibraryPreferences",
+                                 "Library",
```
Comment (ederign) [inline drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java]: done

File: drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java [hunk 4]
```diff
@@ -80,6 +95,11 @@ public void setupMenu() {
                     }
                 } )
                 .endMenu()
+                .newTopLevelMenu( MenuFactory.newSimpleItem( "Admin - Preferences" )
```
Comment (ederign) [inline drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java]: done

File: drools-wb-webapp/src/main/java/org/drools/workbench/client/perspectives/AuthoringPerspective.java [hunk 5]
```diff
@@ -166,4 +174,10 @@ public void execute() {
         return menuItems;
     }
 
+    public void onLibraryContextSwitchEvent( @Observes final LibraryContextSwitchEvent event ) {
+        if ( event.isProjectFromExample() ) {
```
Comment (ederign) [inline drools-wb-webapp/src/main/java/org/drools/workbench/client/perspectives/AuthoringPerspective.java]: done

**Comment by ederign** [comment]
jenkis retest this

**Comment by ederign** [comment]
jenkins retest this

**Comment by ederign** [comment]
2 mins to fix this.

**Pull request merged**